### ISSUE TITLE
TF-GPU in fiji but CPU is max out

### ISSUE THREAD

**Issue by XarlesSta** [juglab/n2v#86]
I am using N2V in Fiji. I installed properly tensor flow. But when I run N2V in Fiji, it seems that the PC is using the CPU and not the GPU. CPU is max out. It seems that Fiji is using the CPU and not the GPU. The Fiji console shows as is Fiji is using GPU-TF and not CPU-TF (see below). Thus, I do not understand why my CPU is max out.  am I missing something here ?

Finally, why is not TF 1.15.0 GPU  available in Edit>Option>TensorFlow ? I had to use the  TF 1.14.0 GPU because I did not see the option for the 1.15 version. I am using a PC with Win10. 

From Fiji-console:
[INFO] Load TensorFlow..
[INFO] Using native TensorFlow version: TF 1.14.0 GPU (CUDA 10.0, CuDNN >= 7.4.1)
Using 10% of training data for validation
[INFO] Tile training and validation data..
[INFO] Generated 200 tiles of shape [128, 128]
[INFO] Create session..
[INFO] Import graph..
[INFO] Normalizing..
[INFO] mean: 253.40125
[INFO] stdDev: 40.8419
[INFO] Augment tiles..
[INFO] Prepare training batches...
65 blind-spots will be generated per training patch of size [64, 64].
[INFO] Prepare validation batches..
65 blind-spots will be generated per training patch of size [64, 64].
[INFO] Start training..
[INFO] Epoch 1/300 
1 / 200 [----------] - loss: 1.836171 mse: 1.836171 abs: 1.108038 lr: 0.000400
2 / 200 [----------] - loss: 1.304834 mse: 1.304835 abs: 0.941930 lr: 0.000400
3 / 200 [----------] - loss: 1.094565 mse: 1.094565 abs: 0.839286 lr: 0.000400
4 / 200 [----------] - loss: 1.069823 mse: 1.069823 abs: 0.795247 lr: 0.000400
...

**Comment by XarlesSta** [comment]
Note: Even when the Fiji console was not showing any "error", N2V was not running on GPUs. It began working after I downloaded, copied, and pasted the proper cuDNN files in the proper directories. 
https://docs.nvidia.com/deeplearning/sdk/cudnn-install/index.html#installwindows

**Comment by tibuch** [comment]
Hi @XarlesSta 

Thank you for reporting this issue. I will move it to the N2V Fiji repository and mention you there as well.
